### ISSUE TITLE
Show detached HEAD in graph

### ISSUE THREAD

**Issue by bradlitterell** [mhutchie/vscode-git-graph#678]
**Describe the feature that you'd like**
HEAD is not available in the branches list.  As a result, when operating in a detached HEAD state, the HEAD does not appear in the graph if it is not also a commit that can be reached as an existing branch.  

However, a commit from this state will result in the graph no longer showing the detached HEAD and latest commit

**Additional context (optional)**
For example, checkout the current "x" branch detached and the commit is visible in the tree with the commit text in bold, but the branch name unbolded.  For example:
![image](https://user-images.githubusercontent.com/49081806/181068377-cb37d0c9-cfdc-4d65-83dd-19139a74f9df.png)

after committing on top of detached head:
![image](https://user-images.githubusercontent.com/49081806/181069048-bdae6b69-d950-46e3-ae59-8c5db9c5ab5c.png)

**Comment by Timmmm** [comment]
I think the name should be bold and the `x` should be `HEAD`.

**Comment by bradlitterell** [comment]
I'm not sure I understand?  The name in this case ("x") is the branch name, but the branch isn't checked out, the commit is.

e.g. git status reports HEAD is detached.

c:\dev\libraries>git status
HEAD detached from 43ae4df6

**Comment by Timmmm** [comment]
Yeah sorry I misunderstood. I think the issue I've found is that if you have a detached head (heh) with uncommited changes then the commit isn't bold and it's quite hard to find the HEAD commit. The only indication is the little dot, which I didn't actually notice until I looked at the code.

I have implemented a "scroll to HEAD" feature which I think is pretty useful and makes it easier to find the current commit. It's useful even when you're on a branch if it's quite far back in history.

Would you be interested in a PR?
